worth2/main/views.py

```diff
@@ -128,6 +129,12 @@ class ParticipantSessionPageView(PageView):
     """WORTH version of pagetree's PageView"""
     gated = True
 
+    def gate_check(self, user):
```

Comment:
Perhaps this could be handled with a login_required decorator on dispatch rather than overriding gate_check?